### DISCUSSION TITLE
升级版本后，提示“字数超出限制”

### DISCUSSION THREAD

**Discussion by kevingekun** [nextai-translator/nextai-translator#1355] [Q&A]
<img width="620" alt="image" src="https://github.com/openai-translator/openai-translator/assets/11899674/02d39de1-562e-419f-8233-0b8ec07c60df">

**Comment by kevingekun** [comment]
使用的是 azure 的服务，gpt3.5

**Reply by mingchaowen** [reply]
之前有个版本遇到了这个问题，后来更新后正常了，今天更新最新版本又出现了。

**Reply by mingchaowen** [reply]
我似乎找到问题的原因了，API URL路径这里应该是这样 `/openai/deployments/xxx/chat/completions`而不是`/openai/deployments/xxx/completions`

**Reply by ricky52529** [reply]
我也遇到了，还真是需要加个 chat，问题解决，你怎么发现需要加 chat 的？

**Reply by mingchaowen** [reply]
> 我也遇到了，还真是需要加个 chat，问题解决，你怎么发现需要加 chat 的？

看的azure openai api文档，https://learn.microsoft.com/zh-cn/azure/ai-services/openai/reference
里面有Completions和Chat Completions
